### PR TITLE
[MetaSchedule][Test] Add unittests for GRP

### PR DESCRIPTION
**Pull request by junrushao1994**
CC: @zxybazh @MasterJH5574

### CODE REVIEW

**Review by MasterJH5574 — APPROVED** [review]
LGTM! Haven't seen new unit tests for long time 😆

File: tests/python/unittest/test_meta_schedule_space_cpu.py [hunk 1]
```diff
@@ -1201,6 +1201,180 @@ def gmm_2(X: T.Buffer[(1, 128, 128), "float32"], Y: T.Buffer[(1, 128, 128), "flo
...
+                    for ax0, ax1, ax2, ax3 in T.grid(1, 1, 28, 64):
+                        with T.block("conv2d_nhwc_global"):
+                            v0 = T.axis.spatial(1, ax0)
+                            v1 = T.axis.spatial(28, i1_0 * 4 + i1_1 + ax1)
+                            v2 = T.axis.spatial(28, ax2)
+                            v3 = T.axis.spatial(128, i3_0 * 64 + ax3)
+                            T.reads(conv2d_nhwc_global[v0, v1, v2, v3])
+                            T.writes(conv2d_nhwc[v0, v1, v2, v3])
+                            conv2d_nhwc[v0, v1, v2, v3] = conv2d_nhwc_global[v0, v1, v2, v3]
+    @T.prim_func
```
Comment (MasterJH5574) [inline tests/python/unittest/test_meta_schedule_space_cpu.py:1252]: Just out of curiosity, do we need blank lines in between 👀?

Comment (junrushao1994) [reply tests/python/unittest/test_meta_schedule_space_cpu.py:1252]: it's probably not a problem here, but if those functions are in global scope, i would prefer adding blank lines in-between

**Comment by junrushao1994** [comment]
@MasterJH5574 Yeah I've been planning to cover the search space of common workloads so that it will be less likely to go wrong when mainline keeps updating (it happened multiple times previously which was quite painful)

**Pull request merged**